plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -64,7 +64,7 @@ type SchedulerServer struct {
 func NewSchedulerServer() *SchedulerServer {
 	s := SchedulerServer{
 		Port:              ports.SchedulerPort,
-		Address:           net.ParseIP("127.0.0.1"),
+		Address:           net.ParseIP("0.0.0.0"),
```

Comment:
Likewise for scheduler. If only status type things are visible, hopefully this is OK.

@stephenR -- opinions?